[FEATURE] Clarify how to update jayvee interpreter in docs

## User Story
1. As a {user}
2. I want {to have clear docs for updating the jv interpreter}
3. So that {I can deal with any errors that might come up}

## User Acceptance Criteria
- [ ] Documentation is updated with a description how to update the interpreter

## Notes
Update done in the CLI, might be getting an EEXIST error only because the version is the same. Will update after rechecking with a new version.

```
pheltweg@rhaznmac2 % npm install -g @jvalue/jayvee-interpreter
npm WARN deprecated @npmcli/move-file@1.1.2: This functionality has been moved to @npmcli/fs
npm ERR! code EEXIST
npm ERR! path /Users/pheltweg/.nvm/versions/node/v18.15.0/bin/jv
npm ERR! EEXIST: file already exists
npm ERR! File exists: /Users/pheltweg/.nvm/versions/node/v18.15.0/bin/jv
npm ERR! Remove the existing file and try again, or run npm
npm ERR! with --force to overwrite files recklessly.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/pheltweg/.npm/_logs/2023-04-28T08_43_54_639Z-debug-0.log
(env) pheltweg@rhaznmac2 % npm install -g @jvalue/jayvee-interpreter --force
npm WARN using --force Recommended protections disabled.
npm WARN deprecated @npmcli/move-file@1.1.2: This functionality has been moved to @npmcli/fs

added 188 packages in 3s

21 packages are looking for funding
  run `npm fund` for details

## Definitions of Done
- [ ] A PR has been opened and accepted
- [ ] All user acceptance criteria are met
- [ ] All tests are passing

--- a/apps/docs/docs/user/intro.md
+++ b/apps/docs/docs/user/intro.md
@@ -14,6 +14,19 @@ Install the interpreter via `npm`. You will need a **nodejs version >= 17.0.0**.
 npm install -g @jvalue/jayvee-interpreter
 ```
 
+You can install a specific version using the `@`-syntax, e.g., version `0.0.16`:
+```bash
+npm install -g @jvalue/jayvee-interpreter@0.0.16
+```
+
+## Update
+
+Updating the interpreter is done by reinstalling it using `npm`. Make sure to also update the [VSCode plugin](#vscode-plugin) to match the installed interpreter if you use it.
+
+```bash
+npm install -g @jvalue/jayvee-interpreter
+```
+
 ## Usage
 
 ### Show help

